code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -392,15 +394,13 @@
 
 /obj/item/modular_computer/update_overlays()
 	. = ..()
-	var/init_icon = initial(icon)
-	if(!init_icon)
-		return
+	var/ui_overlay = icon_overlays ? icon_overlays : initial(icon)
```

Comment:
This should be a var on programs instead of tablets

Reply:
Actually no, I suppose this is fine. Overwriting the icon from the program would put the icon of tablets as the icon for computers and laptops.